src/app/components/courses-page/breadcrumbs/breadcrumbs.component.scss

```diff
@@ -0,0 +1,20 @@
+ @use '../../../../styles/variables' as *;
+
+nav {
+    background-color: map-get($colors, 'white');
+}
+
+.list {
+      list-style-type: none;
+      margin: 0;
+      padding-left: 10;
+      padding-top: 15px;
+      padding-bottom: 15px;
```

Comment:
You may combine all padding in one property: padding: 15px 10px 15px 0;

Reply:
Yes, you are right. But in my case: padding: 15px 0 15px 10px;
And I did a mistake with padding-left: 10 (forgot px)!